pmd-swift/src/main/java/net/sourceforge/pmd/lang/swift/SwiftLanguageModule.java

```diff
@@ -30,7 +30,7 @@ public SwiftLanguageModule() {
                               .addVersion("5.4")
                               .addVersion("5.5")
                               .addVersion("5.6")
-                              .addDefaultVersion("5.7"),
+                              .addDefaultVersion("5.9"),
```

Comment:
5.7 is still supported even if no longer the default
```suggestion
                              .addVersion("5.7")
                              .addDefaultVersion("5.9"),
```

Reply:
yes, but there is no Swift5.8.swift file to test

Reply:
and 5.7 of course